motoman_msgs/srv/SelectTool.srv

```diff
@@ -0,0 +1,92 @@
+
+# Changes the tool file used by the controller for (Moto)ROS-controlled motions.
+#
+# NOTE 1: this changes the active tool for (Moto)ROS-controlled motions ONLY.
+#
+#         Regular INFORM programs, jogging or any other (programmed) motions
```

Comment:
This note is not valid if #440 gets merged.

Reply:
True, but see https://github.com/ros-industrial/motoman/pull/440#discussion_r767944176.

I'm not sure, but I believe @EricMarcil would rather we do not add `mpSetToolNo(..)`.

I added this `NOTE` assuming that was what we were going to end up with.

Reply:
I've discussed it with Ted and we both agreed to put it in.  So we can specify that the current jogging tool will also be changed.

Reply:
Ok. I've updated the `.srv` docs to reflect this.